packages/opentelemetry-plugin-ioredis/src/utils.ts

```diff
@@ -0,0 +1,88 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ioredisTypes from 'ioredis';
+import { Tracer, SpanKind, Span, CanonicalCode } from '@opentelemetry/types';
+import { IORedisPluginClientTypes, IORedisCommand } from './types';
+import { IORedisPlugin } from './ioredis';
+import { AttributeNames } from './enums';
+
+const endSpan = (span: Span, err: Error | null) => {
+  if (err) {
+    span.setStatus({
+      code: CanonicalCode.UNKNOWN,
+      message: err.message,
+    });
+  } else {
+    span.setStatus({ code: CanonicalCode.OK });
+  }
+  span.end();
+};
+
+export const getTracedSendCommand = (tracer: Tracer, original: Function) => {
+  return function sendCommandTrace(
+    this: ioredisTypes.Redis & IORedisPluginClientTypes,
+    cmd?: IORedisCommand
+  ) {
+    const parentSpan = tracer.getCurrentSpan();
+
+    if (arguments.length === 1 && typeof cmd === 'object') {
+      const span = tracer.startSpan(cmd.name, {
+        kind: SpanKind.CLIENT,
+        // Change || to ?? once widely supported in IDEs
+        parent: parentSpan || undefined,
```

Comment:
I believe with the change introduced in https://github.com/open-telemetry/opentelemetry-js/pull/569 we can remove the `|| undefined`

Reply:
True, but I get a type error when I remove it, Is it because this packages uses `"@opentelemetry/types": "^0.2.0"` ? If so I will leave a note to remove once we update to the `0.3.0` types

Reply:
Can't you just bump it with lerna ?

Reply:
lerna should link the packages together. run `lerna run compile` to compile the latest versions of dependencies.

Reply:
Thanks guys `lerna bootstrap` did for me